docker/Dockerfile

```diff
@@ -0,0 +1,31 @@
+## First stage: build the application
+FROM golang:1.13.0-alpine AS builder
+ARG GOARCH
+ARG GOARM
+
+WORKDIR /go/src/github.com/CESARBR/knot-babeltower
+COPY . .
+
+# Install glide dependency manager
+RUN apk update \
```

Comment:
Is `update` really necessary? Couldn't we just use `add` instead? :eyes:

Reply:
Yes. The [link](https://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management#Update_the_Package_list) you recommended says that. The question is: does `--update` option the same thing? I'll test it and update it if necessary.

Reply:
Done.